### ISSUE TITLE
Focus not moved to and trapped in popovers

### ISSUE THREAD

**Issue by st3iny** [nextcloud/nextcloud-vue#2860]
Once a popover is opened the focus should be moved inside it (similar to how it's done with modals). Additionally, the focus should be trapped inside a popover.

**Comment by PVince81** [comment]
work was done for this for dialogs already, probably the same techniques need to be added for popovers also

see https://github.com/nextcloud/nextcloud-vue/pull/2654
